### PR TITLE
[use case] Noise Suppression

### PR DESCRIPTION
**Pull request by ibelem**
@anssiko Thanks for improving the WebNN Spec, for use cases we may lack of some NLP or RNN use cases, for example the Noise Suppression becomes popular than before. PTAL.

***
<a href="https://pr-preview.s3.amazonaws.com/ibelem/webnn/pull/61.html" title="Last updated on May 29, 2020, 8:26 AM UTC (3a8883d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/webmachinelearning/webnn/61/1bed149...ibelem:3a8883d.html" title="Last updated on May 29, 2020, 8:26 AM UTC (3a8883d)">Diff</a>

### CODE REVIEW

**Comment by huningxin** [comment]
Thanks @ibelem for contributing this use case.

According to [resolution 02](https://www.w3.org/2020/05/28-webmachinelearning-minutes.html#r02) of [WebML CG Teleconference – 28 May 2020](https://www.w3.org/2020/05/28-webmachinelearning-minutes.html)

> RESOLUTION: Adopt the noise suppression use case, evaluate rnnoise model for required ops

this PR is going to be merged. 

@ibelem , could you please solve the conflicts before that?

**Comment by ibelem** [comment]
Thanks @huningxin, the conflict should be resolved, PTAL.

File: index.html [hunk 1]
```diff
@@ -1221,9 +1221,12 @@
 		}
 	}
 </style>
-  <meta content="Bikeshed version 70ffa934, updated Fri May 1 10:12:04 2020 -0700" name="generator">
+  <meta content="Bikeshed version e3f9ab1a, updated Fri Apr 24 17:43:54 2020 -0700" name="generator">
   <link href="https://webmachinelearning.github.io/webnn/" rel="canonical">
+<<<<<<< HEAD
+=======
   <meta content="c14010b8fa65f7d1e200a095234fd2ea431c9468" name="document-revision">
+>>>>>>> upstream/master
```
Comment (huningxin) [inline index.html:1229]: Please solve this conflict message.

File: index.html [hunk 2]
```diff
@@ -1470,7 +1473,11 @@
   <div class="head">
    <p data-fill-with="logo"></p>
    <h1 class="p-name no-ref" id="title">Web Neural Network API</h1>
+<<<<<<< HEAD
+   <h2 class="no-num no-toc no-ref heading settled" id="subtitle"><span class="content">Draft Community Group Report, <time class="dt-updated" datetime="2020-05-18">18 May 2020</time></span></h2>
+=======
    <h2 class="no-num no-toc no-ref heading settled" id="subtitle"><span class="content">Draft Community Group Report, <time class="dt-updated" datetime="2020-05-20">20 May 2020</time></span></h2>
+>>>>>>> upstream/master
```
Comment (huningxin) [inline index.html:1480]: ditto.

File: index.html (hunk 1 above)
Comment (ibelem) [reply index.html]: Should be fixed, PTAL. @huningxin

File: index.html (hunk 2 above)
Comment (ibelem) [reply index.html]: Should be fixed, PTAL. @huningxin

**Comment by ibelem** [comment]
The conflict should be resolved, @huningxin PTAL.

**Comment by huningxin** [comment]
Thanks @ibelem , I am going to merge it.

**Comment by ibelem** [comment]
@huningxin Many thanks!

**Pull request merged**